modules/andes-core/broker/src/main/java/org/wso2/andes/kernel/subscription/AndesSubscription.java

```diff
@@ -246,7 +246,7 @@ public void onMessageAck(long messageID) throws AndesException {
      * @throws AndesException on a message re-schedule issue
      * @return DeliverableAndesMetadata reference of rejected message
      */
-    public synchronized DeliverableAndesMetadata onMessageReject(long messageID, boolean reQueue)
```

Comment:
It was required ealrier because  different outbound message trackers were being used in each invocation earlier. Now that the same is used, synchronication is no longer required.